guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -184,4 +185,7 @@ endif::[]
 # {foreman-installer}
 ----
 endif::[]
+ifdef::satellite[]
+. If you are using an external database, upgrade to PostgreSQL 13.
+endif::[]
```

Comment:
This needs to happen before `satellite-maintain upgrade check`, as that one will already complain about the wrong version.

Reply:
Should I just move this to the end for both Foreman and Satellite @evgeni ?

Reply:
Twice in the same procedure? No.
But Foreman/Katello and Satellite procedures *are* quite different, and each of them needs that step.